src/metadata/module_builder.rs

```diff
@@ -148,6 +148,8 @@ impl<'a> ModuleBuilder<'a> {
                 self.build_optional_field_map(builder.init_field_map(), field_map);
             }
 
+            ValueConstructorVariant::Inline => {}
```

Comment:
```suggestion
            // TODO: serialize
            ValueConstructorVariant::Inline => {}
```

We will need to handle this for the feature I'm working on currently. I can handle this after merging.